### ISSUE TITLE
Schematics appearance

### ISSUE THREAD

**Issue by VojislavM** [GliaX/pulseox#22]
It will be good to rearrange components and make schematics more readable. Then it will be more appealing to the people, and it can be used as documentation. If you have some notes about the schematics, please write them on the .sch sheet. 
Regarding the documentation, it is a good practice to have schematics also in PDF version.

**Comment by tareko** [comment]
@Kliment can comment on this. As discussed with @slomusti, happy to have you redraw the schematics in KiCad as you see fit. Probably no need to do serious PCB work beyond that at this point until the device has been tested and firmware written.

**Comment by VojislavM** [comment]
I agree, first, we need to test the boards. In the meantime, I will do the schematics in KiCad.
